loom/common/core/performance.cpp

```diff
@@ -233,14 +233,6 @@ void LoomProfiler::reset()
 
 LoomProfilerRoot::LoomProfilerRoot(const char *name)
 {
-    for (LoomProfilerRoot *walk = sRootList; walk; walk = walk->mNextRoot)
```

Comment:
Maybe. Due to how the new system reuses roots through this exact same search below, we shouldn't get duplicates unless we add some additional code that creates new roots manually.